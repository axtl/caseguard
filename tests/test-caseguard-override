#!/bin/sh

"$TESTDIR/hghave" caseguard || exit 80

echo '% override test'
hg init repo3
cat << EOF >> $HGRCPATH
[extensions]
caseguard = 
EOF
cd repo3

# simple case collision
echo com1 > com1
hg add -w com1 || echo '% disable windows check failure'
echo bar > bar
hg add bar
mv bar BAR
hg add -o BAR || echo '% disable collision check failure'
mv BAR Bar
echo aux > aux
hg add -w
hg add -o

hg st

cd ..
rm -rf repo3