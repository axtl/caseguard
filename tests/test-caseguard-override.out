% override test
abort: case-collision danger
adding Bar
adding aux
A BAR
A Bar
A aux
A bar
A com1
